frontend/src/components/APIService.js

```diff
@@ -0,0 +1,71 @@
+export default class APIService {
+  static LoginUser(body) {
```

Comment:
you rather should pass exact items with their names instead of `body` which doesn't say what it is

e.g.: `(login, password)` or `({ login, password })`